src/main/java/com/gmail/goosius/siegewar/playeractions/PlaceBlock.java

```diff
@@ -355,8 +357,16 @@ private static void evaluatePlaceChest(Player player, Block block) throws TownyE
 		if(adjacentSieges.size() > 1)
 			throw new TownyException(translator.of("msg_err_siege_war_too_many_adjacent_towns"));
 
-		//Attempt plunder.
-		PlunderTown.processPlunderTownRequest(player, adjacentSieges.iterator().next());
+		Siege adjacentSiege = adjacentSieges.iterator().next();
+
+		//Attempt plunder. Call TownPlunderedEvent if the plunder doesn't throw an exception.
+		try {
+			PlunderTown.processPlunderTownRequest(player, adjacentSiege);
+		} catch (TownyException e) {
+			return;
+		}
+
```

Comment:
```suggestion
		//Attempt plunder.
		PlunderTown.processPlunderTownRequest(player, adjacentSieges.iterator().next());

		//Call TownPlunderedEvent if the plunder doesn't throw an exception.
```

Reply:
Catching and ignoring the TownyException would have black-holed the error message feedback.